uspv/init.go

```diff
@@ -137,7 +120,46 @@ func (s *SPVCon) Connect(remoteNode string) error {
 		return err
 	}
 	s.WBytes += uint64(n)
+	return nil
+}
+
+// Connect dials out and connects to full nodes.
+func (s *SPVCon) Connect(remoteNode string) error {
+	var err error
+	var listOfNodes []string
+	if lnutil.YupString(remoteNode) {
+		// if remoteNode is "yes" but no IP specified, use DNS seed
+		listOfNodes, err = s.GetListOfNodes()
+		if err != nil {
+			return err
+		}
+	} else { // else connect to user-specified node
+		if !strings.Contains(remoteNode, ":") {
+			remoteNode = remoteNode + ":" + s.Param.DefaultPort
+		}
+		listOfNodes = []string{remoteNode}
+	}
 
+	for len(listOfNodes) != 0 {
+		err = s.DialNode(listOfNodes)
+		if err != nil {
```

Comment:
display something here so the user can know what's going on; this could be the address they supplied failing, or lots of DNS seed hosts not answering and could take a while

Reply:
cool